### DISCUSSION TITLE
Weather Widget

### DISCUSSION THREAD

**Discussion by APT256** [Lissy93/dashy#806] [Hot Tips]
I was having issues getting the Weather Widget to display info for my city (Decatur, AL). If I tried using Decatur,US, there are 5 such cities and the first one (Decatur, IL) is displayed. If I tried Decatur,AL, I would get an error. I talked to an OpenWeatherMap SE, and he told me to try Decatur,AL,US. This works! Could we please get this added to the Widgets Documentation?

**Comment by lissy93** [comment]
I think it's already in the docs. See [weather widget 401](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#weather-forecast-widget-401).

But if you find things like this which can be improved on, please please do submit a pull request!!
